ray_julia_jll/deps/build.jl

```diff
@@ -20,7 +20,8 @@ open(joinpath(build_dir, "WORKSPACE.bazel"), "w+") do io
     Mustache.render(io, template, dict)
 end
 
-isdir(ray_dir) || cd(dirname(ray_dir)) do
+# Clone "ray" repo when the directory is missing or empty
+isdir(ray_dir) && !isempty(readdir(ray_dir)) || cd(dirname(ray_dir)) do
```

Comment:
The `Dockerfile` doesn't currently utilize this new feature but this was handy when I was precreating the `ray_julia_jll/deps/ray` directory with a symlink.